ui/admin/app/templates/scopes/scope/workers/index.hbs

```diff
@@ -34,64 +34,46 @@
 
   <page.body class='workers'>
     {{#if @model}}
-      <Rose::Toolbar as |toolbar|>
-        <toolbar.primary>
-          <Rose::Form as |form|>
-            {{#let
-              (resource-filter 'scopes.scope.workers.index' 'tags')
-              as |filter|
-            }}
-              {{#if filter.allowedValues}}
-                <Rose::Dropdown
-                  @text={{t (concat 'form.' filter.name '.label')}}
-                  @count={{filter.selectedValue.length}}
-                  @ignoreClickInside={{true}}
-                  as |dropdown|
-                >
-                  {{#each-in
-                    (group-by 'key' filter.allowedValues)
-                    as |key tags|
-                  }}
-                    <dropdown.section @title={{key}} @icon='tag' />
-                    <form.checkboxGroup
-                      @name={{key}}
-                      @items={{tags}}
-                      @selectedItems={{filter.selectedValue}}
-                      @onChange={{fn this.callFilterBy filter.name}}
-                      @itemEqualityFunc={{this.isEqual}}
-                      as |group|
-                    >
-                      <dropdown.item class='indent-label-1'>
-                        <group.checkbox
-                          @label={{group.item.value}}
-                          value={{group.item}}
-                        />
-                      </dropdown.item>
-                    </form.checkboxGroup>
-                  {{/each-in}}
-                </Rose::Dropdown>
-              {{/if}}
-            {{/let}}
-            {{#if
-              (has-resource-filter-selections
-                'scopes.scope.workers.index' 'tags'
-              )
-            }}
-              <Hds::Button
-                @size='small'
-                @icon='x'
-                @text={{t 'actions.clear-all-filters'}}
-                {{on 'click' this.callClearAllFilters}}
-              />
-            {{/if}}
-          </Rose::Form>
-        </toolbar.primary>
-
-        <toolbar.secondary>
-          <ToolbarRefresher @onClick={{this.refresh}} />
-        </toolbar.secondary>
 
-      </Rose::Toolbar>
+      <div class='workers-filtering'>
+        <Dropdown
+          name='tags'
+          @listPosition='bottom-left'
+          @toggleText={{t 'form.tags.label'}}
+          @itemOptions={{this.groupedTags}}
+          @checkedItems={{this.tags}}
+          @applyFilter={{fn this.applyFilter 'tags'}}
+          @isGrouped={{true}}
+          as |FD selectItem itemOptions|
+        >
+          {{#each-in (group-by 'key' (decode itemOptions)) as |key groupItems|}}
```

Comment:
Do you need to decode this? It looks like you just end up using the encoded values anyways with `item`?

Reply:
yes, this needs to be decoded as we pass the `Id` and the `name` to the filters 
<img width="636" alt="Screenshot 2025-05-12 at 2 21 49 PM" src="https://github.com/user-attachments/assets/7c7a6f36-cb46-46ef-bbd1-2df5c94bbb17" />


to display the key in the dropdown we will need to decode the `id`

Reply:
I'm not sure I understand, you have the `name` right there in the object to use for the dropdown, it's still not clear to me why you'd need to decode it since everything else would be using the encoded value

Reply:
I missed mentioning that the `key` we display in the dropdown is grouped in the `encodedId`

Reply:
What is `key` in this case? It's not the `name`?

Reply:
Ah sorry I see what you mean, I was focused on the dropdown values and not the grouping separator. I would still personally have either grouped them beforehand so you don't have to decode and re-encode in the template or just have included the original values as part of the objects

Reply:
for example `key`(which is grouped in the encodedId) is `os` and the filter name we display is actually the `value` 
<img width="320" alt="Screenshot 2025-05-12 at 3 59 59 PM" src="https://github.com/user-attachments/assets/424f16e6-66e4-490e-b32a-09106796a92a" />
<img width="538" alt="Screenshot 2025-05-12 at 4 00 15 PM" src="https://github.com/user-attachments/assets/2d127498-5500-4a27-a44d-5237acc98d91" />

the filter expects `id` and a `name` and we can't display the `key` since there could be multiple `values` for the same key

Reply:
Thanks! I like the approach of not having to encode/decode in the template and pass the original and handle these in the route/controller. Updated accordingly